controllers/mysql_conf.go

```diff
@@ -0,0 +1,83 @@
+package controllers
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+var (
+	// Default options of mysqld section
+	defaultMycnf = map[string]string{}
```

Comment:
Please add these.
```
character-set-server = utf8mb4
collation-server = utf8mb4_unicode_ci
skip-character-set-client-handshake
default-time-zone = +0:00
```

ref. 
- https://stackoverflow.com/questions/3513773/change-mysql-default-character-set-to-utf-8-in-my-cnf
- https://github.dev.cybozu.co.jp/hazama/mysql/blob/master/cybozu/resources/etc/opt/cybozu/mysql/base.cnf